Libs/vtkAddon/vtkStreamingVolumeCodec.h

```diff
@@ -111,14 +111,18 @@ class VTK_ADDON_EXPORT vtkStreamingVolumeCodec : public vtkObject
   /// \param parameters Map containing the parameters and values to be set
   virtual void SetParameters(std::map<std::string, std::string> parameters);
 
-  /// Returns a list of the names of the supported parameter presets
+  /// Returns a list of the human readable names of the supported parameter presets
   std::vector<std::string> GetParameterPresetNames() const;
 
-  /// Returns a string representing the specified parameter
-  std::string GetParameterPreset(const std::string& name) const;
+  /// Get the string represeting the preset value given the name of the parameter
+  /// \param presetName String containing the name of the preset
+  /// Returns a string representing the preset value
+  std::string GetParameterPreset(const std::string& presetName) const;
```

Comment:
What do you think about being more explicit and call this "GetParameterPresetValue"?